.github/ISSUE_TEMPLATE/other.md

```
@@ -0,0 +1,17 @@
+---
+name: Other
+about: Questions, documentation issues, discussions, or anything else.
+labels: question
+---
+
+## Category
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)